libs/sdk-ui-filters/src/AttributeFilter/AttributeFilter.tsx

```diff
@@ -202,6 +214,27 @@ class AttributeFilterCore extends React.PureComponent<IAttributeFilterProps, IAt
         }
     };
 
+    private getValidElementsFilters = () => {
+        const { parentFilters, parentFilterOverAttribute } = this.props;
```

Comment:
Can this be extracted and reused? AttributeFilterButton has a very similar method.